misc/slack-notification/on-start.sh

```diff
@@ -10,3 +10,7 @@ MESSAGE="${ACTION} running \`${ENV0_DEPLOYMENT_TYPE}\` deployment."
 
 __dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 source "${__dir}"/slack.sh "$MESSAGE"
+
+# Making sure we don't have the apply file as we need it to decide if we ran apply or not
+touch apply
+rm apply
```

Comment:
```suggestion
rm -f apply
```

+ missing newline

Reply:
i want to `touch` and then delete this file since `rm` throw exception when the file doesn't exists
its either that or `rm -f apply || true`